lametro/wagtail_hooks.py

```diff
@@ -180,8 +186,24 @@ def get_comment_conditions(self, obj):
     get_comment_conditions.short_description = "Comment conditions"
 
 
+class FiscalYearCalendarAdmin(ModelAdmin):
```

Comment:
I experimented with removing the "add fiscal year calendar" button in the cms, similar to what's going on with the board member details, but had some trouble. But since we can prevent metro from adding more by changing their permissions, I figured this wasn't critical.

And even if the button is there, when trying to add another, the `ValidationError` set up in the model's `clean` method displays and prevents creation.